LICENSE

```diff
@@ -0,0 +1,27 @@
+BSD 2-Clause License
+
+Copyright (c) 2014-2022, Tarantool AUTHORS
+Copyright (c) 2014-2017, Dmitry Smal
+Copyright (c) 2014-2017, Yura Sokolov aka funny_falcon
```

Comment:
Oh, Yura Sokolov did not develop the library.

It should be something like:

```suggestion
Copyright (c) 2025, Tarantool AUTHORS
```

Reply:
fixed :(